husky_description/urdf/husky.urdf.xacro

```diff
@@ -9,6 +9,11 @@
 
   <xacro:include filename="$(find husky_description)/urdf/husky_macro.urdf.xacro" />
 
+  <xacro:include filename="$(find velodyne_description)/urdf/VLP-16.urdf.xacro"/>
+  <xacro:VLP-16 parent="lidar_link" name="velodyne" topic="/velodyne_points" organize_cloud="false" hz="10" samples="440" gpu="false">
```

Comment:
2 more things: 
1. Can you change topic to `/points` to align with Ouster.
2. Can you enable GPU by default, else it will run slow for everyone